EnglishTranslation-master/QUEST_LV_0100.tsv

```diff
@@ -11345,6 +11337,7 @@ QUEST_LV_0100_20151224_011336	Well, it's no big issue.{nl}Everything has a reaso
 QUEST_LV_0100_20151224_011337	The Royal Mausoleum must contain a secret bigger than we could possibly imagine.{nl}The next epitaph is at Serno Highland. Let's go.
 QUEST_LV_0100_20151224_011338	As Laulas's injury are severe, I'm here in his stead.{nl}The seal is released, what exactly happened here?{nl}
 QUEST_LV_0100_20151224_011339	Whatever the case may be, the altars here are one of the{nl}Royal Mausoleum's various defenses built to ward off intruders.{nl}They should, however, the seal at Rukas Plateau is the answer.{nl}The altar won't function if it gets destroyed, but if we can activate it before that.{nl}
+QUEST_LV_0100_20151224_011340	

```

Comment:
The `They should, however, the seal at Rukas Plateau is the answer.{nl}The altar won't function if it gets destroyed, but if we can activate it before that` part from 11339 should be in 11340